gregrickaby/gatsby-graphql-wordpress-starter
# Adding StorybookJS to Gatsby Starter Theme

This is still a Work In Progress

## Description

Add StorybookJS to Gatsby Starter, this is still a **Work In Progress** not sure if it suits for master at the moment, would love any feedback, at the moment I have add 3 components, `Header, Footer & Byline`

## Screenshot

<img width="815" alt="Screen Shot 2019-12-03 at 13 40 07" src="https://user-images.githubusercontent.com/3707088/70083821-bf003a00-15d2-11ea-955c-8fa99dbf0058.png">

## Steps for testing

Install the CLI `npm install -g @storybook/cli`

Install the packages with `yarn`

Then run  `npm run storybook`

got to `localhost:6006/` to see your storybook
